src/citation_vs_reviewscore.py

```diff
@@ -0,0 +1,15 @@
+import csv
+import os
+import sys
+
+src_path = os.getcwd()
+data_path = src_path.strip('src') + 'data/'
+input_file = 'acta_a_citations_review_scores_utf8.csv'
```

Comment:
static global variables we usually give a name in all caps.  I would put it all the way at the top (below the imports) and separated from the the rest of the code by a line as you may want to in and edit the name without changing any other code if you want to read a different file.  it is easy to find. Pick a variable name that makes sense too. `input_file` is good but `INPUT_FILE_NAME` might be better because it takes a string (i.e., the name not a file object).   These are small things and don't matter in the functioning of the code, but help a lot with readability/maintainability

Reply:
Indeed, I would like to improve on proper 'pythonic' practice. So, I really appreciate stuff like this!